Specta/Specta/SPTExcludeGlobalBeforeAfterEach.h

```diff
@@ -3,8 +3,8 @@
  */
 #import <Foundation/Foundation.h>
 
-// This protocol is used for blacklisting classes for global beforeEach and afterEach blocks.
-// If you do not want a class to participate in those just add this protocol to a category and it will be
-// excluded.
+// This protocol was used for blacklisting classes for global beforeEach and afterEach blocks.
+// Now, instead, classes are whitelisted by implementing the SPTIncludeGlobalBeforeAfterEach protocol.
+__deprecated
```

Comment:
@orta, what are your thoughts on deprecating this vs. removing it altogether? In comments on #148 you seemed gung-ho for moving fast, breaking things, and bumping Specta to 1.0.